package/yast2-mail.spec

```diff
@@ -17,46 +17,33 @@
 
 
 Name:           yast2-mail
-Version:        3.1.9
+Version:        3.1.10
 Release:        0
 
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Source0:        %{name}-%{version}.tar.bz2
 
-
-Group:	        System/YaST
-License:        GPL-2.0+
-BuildRequires:	perl-XML-Writer update-desktop-files yast2-testsuite
+BuildRequires:  update-desktop-files
+BuildRequires:  yast2
 BuildRequires:  yast2-devtools >= 3.1.10
-BuildRequires:	yast2-auth-server yast2-network yast2-perl-bindings yast2-users
-BuildRequires:  perl-NetxAP
+BuildRequires:  yast2-testsuite
 
 PreReq:         %fillup_prereq
 
 # new firewall interface
 # Wizard::SetDesktopTitleAndIcon
 # cfg_mail.scr
-Requires:	yast2 >= 3.1.179
-Requires:	yast2-users
-Requires:	yast2-ldap
-Provides:	yast2-config-network:/usr/lib/YaST2/clients/lan_sendmail.ycp
-Provides:	yast2-config-sendmail yast2-config-sendmail-devel
-Obsoletes:	yast2-config-sendmail yast2-config-sendmail-devel
-Provides:       yast2-trans-sendmail = %{version}
-Obsoletes:      yast2-trans-sendmail <= %{version}
-Provides:       yast2-config-mail = %{version}
-Obsoletes:      yast2-config-mail <= %{version}
-Provides:       yast2-trans-mail = %{version}
-Obsoletes:      yast2-trans-mail <= %{version}
-Provides:       yast2-mail-server = %{version}
-Obsoletes:      yast2-mail-server <= %{version}
-Provides:       yast2-mail-plugins = %{version}
-Obsoletes:      yast2-mail-plugins <= %{version}
-Conflicts:	aaa_base < 10.3
-BuildArch:	noarch
+Requires:       yast2 >= 3.1.179
+Requires:       yast2-ldap
+Requires:       yast2-users
+Provides:       yast2-config-network:/usr/lib/YaST2/clients/lan_sendmail.ycp
+Conflicts:      aaa_base < 10.3
+BuildArch:      noarch
 Requires:       yast2-ruby-bindings >= 1.0.0
 
-Summary:	YaST2 - Mail Configuration
+Summary:        YaST2 - Mail Configuration
+License:        GPL-2.0+
```

Comment:
Don't change the license, there might be code of 3rd parties in YaST (patches from community or partners etc.).
The rule was not to use GPL2+ for new packages, those should get 2-only.